CHANGELOG

```diff
@@ -1,3 +1,14 @@
+snowplow-ecommerce 0.9.2 (2025-06-30)
+---------------------------------------
+## Summary
+This release fixes a bug where specifying passthrough fields was having no effect.
+
+## Features
```

Comment:
A bit NIT but I can see it was "incorrect" in the previous release change log as well: can we add this under Fixes? See example [here](https://github.com/snowplow/dbt-snowplow-unified/blob/ba88b9feb453734cc5f98527ce043826e5cf10ed/CHANGELOG#L23C4-L23C9). It would also be nice to rename the commit to a descriptive sentence instead keeping the PR reference? We will be able to see the internal references from there. Then here this would be the same as the commit name.